domains/api-popcord.json

```diff
@@ -0,0 +1,9 @@
+{
+  "description": "api-popcord.is-a.dev",
+  "owner": {
+    "username": "spidirman",
```

Comment:
Invalid JSON, you should have an email field here, you should also have proper JSON formatting.

Reply:
changed, i think its good